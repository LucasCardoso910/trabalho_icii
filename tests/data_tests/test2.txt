8
95
67
56
45
43
19
12
08