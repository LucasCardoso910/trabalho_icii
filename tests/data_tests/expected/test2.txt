bubble_sort:28,84
heap_sort:25,52
insert_binary_sort:17,42
insert_sort:35,49
merge_sort:12,32
quick_sort:24,29
selection_sort:28,21
shake_sort:28,64
